### PR TITLE
[9.0] open form from map info box

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

File: base_geoengine/README.rst [hunk 1]
```diff
@@ -24,14 +24,22 @@ The module also requires two additional python libs:
 
 for a complete documentation please refer to the `public documenation <http://oca.github.io/geospatial/index.html>`_
 
+Known issues / Roadmap
+======================
+
+ * Google layers have been removed as it was not working anyway.
```
Comment (lmignon) [inline base_geoengine/README.rst:30]: Is it the case in this PR?

**Comment by lmignon** [comment]
LGTM (Code + functionnal) :+1: 
Error is travis is unrelated.

File: base_geoengine/README.rst (hunk 1 above)
Comment (yvaucher) [inline base_geoengine/README.rst:30]: It was done previously I just updated it with what came to my mind.

Comment (lmignon) [inline base_geoengine/README.rst:30]: @yvaucher you can also remove https://github.com/yvaucher/geospatial/blob/ec85e432cf1f078860c4a8d758d2324af61b16ab/base_geoengine/data/geo_raster_layer_type.xml

**Pull request merged**